tests/cpp/unit-tests/test_addressatcommand.cpp

```diff
@@ -0,0 +1,108 @@
+#include "../../../third-party/catch/catch.hpp"
+#include "addressatcommand.h"
+
+using namespace SmartRedis;
+
+SCENARIO("Ensuring the iterators for an AddressAtCommand are correct", "[AddressAtCommand]")
```

Comment:
General comment: Should we also test member variables specific to ``AddressAtcommand`` and ``NonKeyedCommand`` (e.g. port number and host)?